plugin/src/main/java/org/opensearch/ml/processor/MLInferenceSearchResponseProcessor.java

```diff
@@ -14,13 +14,7 @@
 import static org.opensearch.ml.processor.MLInferenceIngestProcessor.OVERRIDE;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Avoid using import *

Reply:
You can just add this in your intellij settings: https://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij

Reply:
reverted  import *